i.py

```diff
@@ -19,7 +19,7 @@
 PRUNING_CHOICES = ['default', 'nothing', 'everything']
 
 MAINNET_VERSION = "22.0.3"
-TESTNET_VERSION = "20.1.0-testnet"
+TESTNET_VERSION = "22.0.0-testnet"
```

Comment:
version should be `22.0.0-rc0` 

https://rpc.testnet.osmosis.zone/abci_info?

Reply:
sorry! fixed